Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -250,8 +251,12 @@ private IBocList GetControlToValidate ()
       return bocListControl;
     }
 
-    void IValidatorWithDynamicErrorMessage.RefreshErrorMessage ()
+    public void RefreshErrorMessage ()
```

Comment:
Consideration: should we find a better name here? Refresh sounds like a can-be-done-many-times API. Maybe just PopulateErrorMessage? GenerateErrorMessage? BuildErrorMessage? -> am tending to BuildErrorMessage when it's one of those three. Better ideas?

Reply:
`BuildErrorMessage` is fine by me